src/analytics/PosthogSpanProcessor.ts

```diff
@@ -157,7 +161,8 @@ export class PosthogSpanProcessor implements SpanProcessor {
   /**
    * Shutdown the processor.
    */
-  public shutdown(): Promise<void> {
-    return Promise.resolve();
+  // eslint-disable-next-line @typescript-eslint/require-await
+  public async shutdown(): Promise<void> {
+    return;
```

Comment:
I would prefer to remove the `eslint-disable` comment and return `Promise.resolve()` in a case like this. It seems more in the spirit of the require-await lint rule.